code/frameworks/nextjs/package.json

```
@@ -87,7 +87,7 @@
   },
   "peerDependencies": {
     "@babel/core": "^7.11.5",
-    "@storybook/addon-actions": "7.0.0-alpha.43",
+    "@storybook/addon-actions": "7.0.0-beta.12",
```

Let's add an `Caret constraint` to make the version not that strict.

```suggestion
    "@storybook/addon-actions": "^7.0.0-beta.12",
```

---

Even tho I'd agree with you, I didn't dare to change it since all the other `@storybook` dependencies are pointing to that same version :D Is it safe to assume that all those deps should follow the same bumping rules, right? Maybe I should bump them as well in order to avoid recreating the conditions that led to this bug? WDYT?

```    "@storybook/builder-webpack5": "7.0.0-beta.12",
    "@storybook/core-common": "7.0.0-beta.12",
    "@storybook/node-logger": "7.0.0-beta.12",
    "@storybook/preset-react-webpack": "7.0.0-beta.12",
    "@storybook/preview-api": "7.0.0-beta.12",
    "@storybook/react": "7.0.0-beta.12",

---

These files should be handled by an automation script by @shilman, it seems like it was a hiccup and he's going to update it

---

The automation script is a bit brittle -- it looks for the current version & then increments it. I think what happened is that the original PR was already out of sync and so the automation never incremented it properly. I'll merge this & verify that it gets incremented on next release.

---

I've updated the release script to deal with `^` versions also per this PR